packages/html/src/tooltip/tests/tooltip-callout-position.tsx

```diff
@@ -0,0 +1,158 @@
+import { AnimationContainer } from '../../animation-container';
+import { TooltipNormal, TooltipClosable } from '../../tooltip';
+
+const style = `
+    #test-area {
+        grid-template-columns: 120px repeat(4, 1fr);
+    }
+
+    .k-animation-container {
+        display: inline-block;
+        position: relative;
+        overflow: visible;
+    }
+
+    .k-tooltip {
+        position: relative;
+    }
+`;
+
+
+export default () =>(
+    <>
+        <style>{style}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-5">
```

Comment:
```suggestion
        <div id="test-area" className="k-d-grid">
```
Those classNames become redundant after applying this **#test-area** style
```css
    #test-area {
        grid-template-columns: 120px repeat(4, 1fr);
    }
``` 
(appears in **tooltip.tsx** and **tooltip-rtl.tsx** as well)